fs/ggml/ggml.go

```diff
@@ -105,32 +105,19 @@ func (kv KV) Bool(key string, defaultValue ...bool) bool {
 }
 
 func (kv KV) Strings(key string, defaultValue ...[]string) []string {
-	r := keyValue(kv, key, &array{})
-	s := make([]string, r.size)
-	for i := range r.size {
-		s[i] = r.values[i].(string)
-	}
+	return keyValue(kv, key, &array[string]{}).values
```

Comment:
is it intentional that `defaultValue` isn't used in any of these array functions? I see it wasn't previously either. I noticed this while rebasing #10238 on top of this

Reply:
no, it wasn't intentional and it's probably a bug. however it doesn't really make sense to have default values for arrays